src/services/API/sqs.js

```diff
@@ -0,0 +1,50 @@
+import { SQSClient, SendMessageCommand } from "@aws-sdk/client-sqs";
+
+const REGION = "ap-south-1";
+const queueURL = "https://sqs.ap-south-1.amazonaws.com/111766607077/EventQueue";
+const sqsClient = new SQSClient({
+  region: REGION,
+  credentials: {
+    accessKeyId: process.env.VUE_APP_AWS_ACCESS_KEY_ID,
+    secretAccessKey: process.env.VUE_APP_AWS_SECRET_KEY,
+  },
+});
+
+async function sendSQSMessage(purpose, purposeParams, redirectTo, redirectID, userID, doesUserExist, authType) {
+    const messageBody = [
+      {
+        dateTime: Date.now().toString(),
+        purpose: {
+          type: purpose,
+          subType: purposeParams,
+          params: {
+            platform: redirectTo,
+            id: redirectID,
+          },
+        },
+        authType: authType,
+        user: {
+          values: userID,
+          userDataValidated: doesUserExist,
```

Comment:
This name mismatch is worrying. Are you checking if the user is validated or whether user exists? I'm guessing `validated` is correct, but then the other variable also should reflect the same